### PR TITLE
BaSP-MR-20: all tests for activities 

### PR DESCRIPTION
**Pull request by IgnacioCanton**
I developed all the tests for the activities. Please review it and give me your feedback

### CODE REVIEW

File: src/tests/activities.test.js [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+  isActive: true,
+};
+
+beforeAll(async () => {
+  await Activity.collection.insertMany(activitiesSeed);
+});
+
 test('should pass a basic test', () => {
   expect(1 + 1).toBe(2);
 });
```
Comment (LucianoStradiot) [inline src/tests/activities.test.js:24]: This should be deleted

Comment (IgnacioCanton) [reply src/tests/activities.test.js]: done

File: src/tests/activities.test.js [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+  name: 'Kick boxing',
+  description: 'Kickboxing is a group of foot combat sports...',
+  isActive: true,
+};
+
+beforeAll(async () => {
+  await Activity.collection.insertMany(activitiesSeed);
+});
+
 test('should pass a basic test', () => {
```
Comment (VictoriaNegrotto) [inline src/tests/activities.test.js]: you should delete this function

**Review by LucianoStradiot — APPROVED** [review]

File: src/tests/activities.test.js (hunk 1 above)
Comment (LucianoStradiot) [reply src/tests/activities.test.js]: AP!

**Review by FontanR — APPROVED** [review]
LGTM

**Pull request merged**